src/tbp/monty/frameworks/loggers/graph_matching_loggers.py

```diff
@@ -412,6 +412,10 @@ def get_formatted_overall_stats(self, mode, episode):
             "episode/confused_mlh": stats["episode_confused_mlh"],
             "episode/pose_time_out": stats["episode_pose_time_out"],
             "episode/time_out": stats["episode_time_out"],
+            "episode/consistent_child_obj": stats["episode_consistent_child_obj"],
+            "episode/consistent_child_or_parent": stats["episode_consistent_child_obj"]
+            or stats["episode_correct"]
+            or stats["episode_correct_mlh"],
```

Comment:
Does this work correctly? It looks wrong to me, usually because Python will assume the end of a statement if there isn't something to indicate that it continues onto the next line. Maybe it works because it's expecting a `,` but the formatting of this looks wrong to me. I'm surprised that `ruff format --check` is okay with this.

Reply:
autoformat broke it down like that to me. We are already doing this on some other lines further up but I can add some brackets just to be safe. Rerunning the experiment again to double check I get the same results.

Reply:
good to know! fwiw it doesn't seem to be any different functionally, I still get the exact same results